zgw/catalogi-api/src/main/kotlin/com/ritense/catalogiapi/CatalogiApiPlugin.kt

```diff
@@ -140,6 +142,27 @@ class CatalogiApiPlugin(
         return results
     }
 
+    fun getBesluitTypes(zaakTypeUrl: URI): List<Besluittype> {
+        var currentPage = 1
+        var currentResults: Page<Besluittype>?
+        val results = mutableListOf<Besluittype>()
+
+        do {
+            logger.debug { "Getting page of besluit types, page $currentPage for zaaktype $zaakTypeUrl" }
+            currentResults = client.getBesluittypen(
```

Comment:
It's `getBesluittypen` here and the method we're in is `getBesluitTypes`. Which of the two is correct?